### PR TITLE
channels/{prerelease,candidate}: Add 4.1.31 and 4.2.15

### PR DESCRIPTION
**Pull request by joepvd**

### CODE REVIEW

**Comment by wking** [comment]
/hold

Grr, I keep forgetting to check CI update success.  Will reply once I have.

**Comment by eparis** [comment]
/lgtm
after trevor pull the hold

**Comment by wking** [comment]
Update testing for [4.1.31][1] and [4.2.15][2] looks good.  Some tests failed on setup (issues with the previous version.  Today mostly issues with [AWS AMI copies][3]).  I've launched replacement jobs for those.  The 4.2.15 release doesn't have a lot of update CI itself, but there's a lot of success in [the source nightly][4] that covers the updates baked into the image metadata.  Waiting on the 4.1.z -> 4.1.31 replacement jobs now, but I'm not particularly worried ([4.1.31][1] looks like a pure rebuild of 4.1.30 with base image changes but no OpenShift changes).

[1]: https://openshift-release.svc.ci.openshift.org/releasestream/4-stable/release/4.1.31#upgrades-from
[2]: https://openshift-release.svc.ci.openshift.org/releasestream/4-stable/release/4.2.15#upgrades-from
[3]: https://bugzilla.redhat.com/show_bug.cgi?id=1752370
[4]: https://openshift-release.svc.ci.openshift.org/releasetag/4.2.0-0.nightly-2020-01-15-231532

**Comment by wking** [comment]
We're going to scrap 4.2.15 because 4.2.16 (with an additional console fix, openshift/console#3991) recycled its errata link:

```console
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.2.16-x86_64 | grep errata
    url: https://access.redhat.com/errata/RHBA-2020:0107
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.2.15-x86_64 | grep errata
    url: https://access.redhat.com/errata/RHBA-2020:0107
```

File: channels/prerelease-4.1.yaml [hunk 1]
```diff
@@ -36,3 +36,4 @@ versions:
 - 4.1.28
 - 4.1.29
 - 4.1.30
+- 4.1.31
```
Comment (sosiouxme) [inline channels/prerelease-4.1.yaml]: https://github.com/openshift/cincinnati-graph-data/pull/27 will handle this individually

File: channels/candidate-4.2.yaml [hunk 2]
```diff
@@ -32,3 +33,4 @@ versions:
 - 4.2.13
 - 4.2.13-s390x
 - 4.2.14
+- 4.2.15
```
Comment (sosiouxme) [inline channels/candidate-4.2.yaml]: this is 4.2.16 now, won't be releasing 4.2.15. only reason to have it here would be to assist in migrating off of it, but i wouldn't think we even want that.

**Comment by sosiouxme** [comment]
I created https://github.com/openshift/cincinnati-graph-data/pull/29 instead

**Comment by wking** [comment]
I've filed #33 with YAML comments about [skipping 4.2.15][1].

[1]: https://github.com/openshift/cincinnati-graph-data/pull/23#issuecomment-575832202

**Pull request closed without merging**